miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppCustomPermissionCache.kt

```diff
@@ -134,3 +166,16 @@ internal class MiniAppCustomPermissionCache(context: Context) {
         return isPermissionGranted
     }
 }
+
+@Suppress("TooGenericExceptionCaught", "SwallowedException")
+internal fun initEncryptedSharedPreference(context: Context) = try {
+    EncryptedSharedPreferences.create(
+        "com.rakuten.tech.mobile.miniapp.custom.permissions.cache.hash",
```

Comment:
This isn't storing hashes, so maybe this should be `com.rakuten.tech.mobile.miniapp.custom.permissions.cache.encrypted`?

Reply:
Yeah, I have updated the parameter.